index.html

```diff
@@ -0,0 +1,62 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>My Portfolio Project</title>
+    <link rel="stylesheet" href="./style.css">
+</head>
+<body>
+    
+        <div class="main-container">
+            <header>
+                <nav class="navbar">
+                    <div class="nav-branding">
+                        <a href="url"><img src="./img/logo-placeholder.png" alt=""></a>
+                    </div>
+                    <div class="nav-menu">
+                        <a href="url"><img src="./img/icon-menu.png" alt=""></a>
+                    </div>
+                </nav>
```

Comment:
- Kindly make sure only the hamburger button has a nav tag. as it was required by the previous reviewer [here](https://github.com/kazim110/My-Portfolio-Project/pull/2#discussion_r1013751719) 

The nav tag is for semantics reasons and the right position for convention and user experience for mobile devices.

The structure must be like this :grinning:  :grinning:  :grinning: 

```
<div class="main-container">
   <header>
             
                    <div class="nav-branding">
                        <a href="url"><img src="./img/logo-placeholder.png" alt=""></a>
                    </div>
                    <nav class="nav-menu">
                        <a href="url"><img src="./img/icon-menu.png" alt=""></a>
                    </nav>
   </header>
```